_pages/security/static-analysis.md

```diff
@@ -16,26 +16,32 @@ There are tools for JS, Ruby, and Python, and you are encouraged to set up this
 Service | Languages/frameworks | Dependency analysis | Code security analysis
 --- | --- | --- | ---
 [Code Climate](https://codeclimate.com/) | Ruby, JS, Python | Ruby and Node (on `push` only) | Rails, Node, and Ember
-[Gemnasium](https://gemnasium.com/) | Ruby, Node, Python, PHP | Y | N
+[Gemnasium](#gemnasium) | Ruby, Node, Python, PHP | Y | N
 [Hakiri](https://hakiri.io/) | Ruby | Y | Rails only
 
 Code analysis can be run locally with the following open source tools. These tools provide results similar (and in some cases, identical) to the hosted services above.
 
-#### Gemnasium setup
+#### Gemnasium
 
-To set up Gemnasium, you'll need to be invited to the devops@gsa.gov account (ask in [#infrastructure](https://gsa-tts.slack.com/messages/infrastructure/)). In this process, be sure to log in via a password (rather than Github login) as that'll tie access to your gsa.gov email (as opposed to personal email).
+##### Setting up your account
+
+1. [Sign into Gemnasium via GitHub.](https://gemnasium.com/login)
+1. Go to [your settings page](https://gemnasium.com/settings) and change it to use your work email, if it isn't already.
```

Comment:
:shrug: we must have missed that the first time around. All I remember is being told explicitly to create a separate account and not log in with Github. If we can't figure out a good reasoning for that, agreed that we should remove.